### DISCUSSION TITLE
DataTable height with sticky header

### DISCUSSION THREAD

**Discussion by makeupsomething** [carbon-design-system/carbon#8491] [Help] [answered]
Hi All 👋 

The current version of the DataTable when used with a stickHeader seems to have a fixed max-height of 300px.
Is there any plan to make this adjustable by the user? 

I'm working on my own implantation where the table height is determined based on the number of rows the user wants to show but I would love to know if there is any other solutions?

**Comment by tw15egan** [comment, marked as answer]
Hey there @makeupsomething, you're definitely free to override the height to suit your needs. I believe a hardcoded value is simply needed to enable the sticky header functionality. By no means are you stuck with having it at 300px. Hope that helps!
